microcosm_flask/conventions/crud.py

```diff
@@ -39,7 +39,7 @@ def configure_search(self, ns, definition):
         :param definition: the endpoint definition
 
         """
-        paginated_list_schema = make_paginated_list_schema(ns, definition.response_schema)()
+        paginated_list_schema = make_paginated_list_schema(ns, definition.response_schema)(strict=True)
```

Comment:
@jessemyers  is the best way to do it? It seems like adding a class properly on the class returned by `make_paginated_list_schema` won't do what I want.